build.py

```diff
@@ -714,10 +714,13 @@ def report_and_check_call(command, *args, cwd=None, shell=False, **kwargs):
         for arg in command:
             print('    {}'.format(repr(arg)))
 
+    if 'check' not in kwargs:
+        kwargs['check'] = True
+
```

Comment:
With the other suggestion, this isn't needed anymore.  But, if it were, or is in the future, it could just be added as a parameter default above similar to how `cwd` is handled.

```suggestion
```